smoke-release-testing/mgconsole/nested_indices.bash

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$SCRIPT_DIR/../utils.bash"
+
+# TODO(gitbuda): Introduce default __host and __port values.
+test_nested_indices() {
+  __host="$1"
+  __port="$2"
+  echo "FEATURE: Nested Indices"
+
+  echo "
+    CREATE (:Product {specifications: {dimensions: {width: 10, height: 20}}});
+    CREATE INDEX ON :Product(specifications.dimensions.width);
+    RETURN 1;
```

Comment:
We should check the output of a creation. You want to run a simple query to see that the index is being hit; this test checks the creation, not that the index is working.